### ISSUE TITLE
Update all open Pull Requests when user install this app.

### ISSUE THREAD

**Issue by sotayamashita** [pr-triage/app#170]
**Additional context**
- https://github.com/wip/app/commit/d6603bbae9ea0d9178d9ff43aec8b809b497f074

**Comment by mishuvs** [comment]
I agree. When I installed the app, I expected it to also have labeled the existing PRs. This should be prioritized.

**Comment by sotayamashita** [comment]
@mishuvs I understand its importance but I do not have time now. I would appreciate if you could create a PR for it. :bow: Is it possible?

**Comment by mishuvs** [comment]
@sotayamashita I don't have any experience with building Probot apps so far and am not sure about when I'll get time to do this, sorry :( 
And I meant it should be prioritized but doesn't have to be done immediately. Adding a priority label or something, in case a new contributor wants to work.
